config/secrets.defaults.yml

```diff
@@ -14,3 +14,9 @@ test:
     secret_key: IBM_CLOUD_OBJECT_STORAGE_SECRET_KEY
   ibm_cloud_object_storage:
     <<: *ibm_cloud_object_storage_defaults
+  iks_defaults: &iks_defaults
+    api_key: IBM_CLOUD_IKS_API_KEY
+    hostname: IBM_CLOUD_IKS_HOSTNAME
```

Comment:
Also just noticed this has to be a valid hostname format (aka no underscores).

Suggest `hostname: ibm-cloud-iks-hostname`